website/content/en/docs/add-ons/storage/efs/guide.md

```diff
@@ -80,7 +91,11 @@ If you prefer to manually setup each component then you can follow this manual g
 export AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query "Account" --output text)
 ```
 
-#### 1. Install the EFS CSI driver
+#### 1. Driver install and IAM configuration
+
+> Important: Terraform deployent users should skip this step.
```

Comment:
Skip this step if you are using Terraform deployment since EFS CSI driver is installed by default unless you set the `install_efs_ = false`

same comment for EFS

Reply:
done